Bottom: 5b63089ab57a050c748aff15d7348e0b139c62fa
Top:    db9448162a8dba984fc26b54e1e51ef22d2da803
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-04-21 02:08:32 +0100

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index ed10b50..79a914a 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
